timemachine/ff/handlers/bcc_aromaticity.py

```diff
@@ -77,7 +78,7 @@ def call_openeye(
         logging.debug(output_string)
 
 
-def match_smirks(smirks: str, oe_molecule: "OEMol", unique: bool = False) -> List[Dict[int, int]]:
```

Comment:
flake8 complains about these being undefined variables, looks like it was decided that this was invalid style: https://github.com/PyCQA/pyflakes/issues/340

Even adding `oechem.OEMol` fails.

Reply:
> Even adding oechem.OEMol fails.

hmm, did you import `oechem`? This seems to work on my end

Reply:
one workaround could be adding `OEMol = Any` at top level, so we can continue to use the annotations for documentation purposes

Reply:
nit: probably fancier than we need, but this looks like a potential solution: https://github.com/PyCQA/pyflakes/issues/340#issuecomment-513821884